src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-manager.context.ts

```diff
@@ -1,8 +1,9 @@
 import { Injectable } from '@angular/core';
 import { SearchComposite } from '../shared/model/search-composite';
+import { GermplasmListModel } from '../shared/germplasm-list/model/germplasm-list.model';
 
 @Injectable()
 export class GermplasmManagerContext {
     searchComposite: SearchComposite<any, number>;
-    sourceListId: number;
+    sourceGermplasmList: GermplasmListModel;
```

Comment:
better to retain sourceListId here (since it's used by multiple screens). In GermplasmListCloneComponent, query the latest list metadata give the sourceListId